app/src/main/kotlin/at/bitfire/davdroid/db/Collection.kt

```diff
@@ -205,6 +209,10 @@ data class Collection(
                 }
             }
 
+            if (timezoneId == null && timezone != null) {
+                timezoneId = DateUtils.parseVTimeZone(timezone).timeZoneId?.value
```

Comment:
We have to catch the Exception here too

Better yet, make it return null if it can't be parsed?